### DISCUSSION TITLE
Workflow receives "This page is taking too long to load" message (with Unicorn!)

### DISCUSSION THREAD

**Discussion by nizzantalon** [community/community#25231] [Actions]
<p>Hello,<br>
I’m unable to load executions in one of my workflows. It worked up until last week and nothing’s changed.</p>
<p>I now get the “Unicorn!” page with the message:</p>
<blockquote>
<p><strong>This page is taking too long to load.</strong></p>
<p>Sorry about that. Please try refreshing and contact us if the problem persists.</p>
</blockquote>
<p>It’s a long workflow (usually runs 90-180 minutes, but I have other similar workflows which run a similar amount and weren’t broken. Unsuccessful executions of the workflow do show up, so not sure if it’s a timing issue or something which is printed later on.</p>
<p>Accessing the workflow with an additional “/workflow” in the URL does manage to display the details:<br>
<a href="https://github.com/XXX/YYY/actions/runs/2019999999/workflow" class="onebox" target="_blank" rel="noopener nofollow ugc">https://github.com/XXX/YYY/actions/runs/2019999999/workflow</a></p>
<p>The workflow runs on a self-hosted Windows machine if it matter.</p>
<p>Anyone else encounter this? Ideas how to resolve? It seems to be an issue on the server-side.</p>

**Comment by sperronc** [comment]
<p>I’m observing the same issue, which has also shown up in the last week (worked fine before). Some workflow runs show up correctly, some not.</p>
<p>Adding the “workflow” in the URL doesn’t work for me.</p>
<p>Really strange.</p>

**Comment by sperronc** [comment]
<p>Update:</p>
<ul>
<li>While workflow is running, access to the workflow works fine. As soon as the last job is executed, we can’t access anymore</li>
<li>We worked around the issue by splitting the workflow into smaller workflows. Not great because it creates many workflows, but it seems to work for now.</li>
</ul>

**Comment by nizzantalon** [comment]
<p>Great to hear we’re not the only ones affected. We currently still haven’t found a solution and splitting our workflow is unfortunately not so simple (would require a few gb of artifcats for each flow which gets very expensive).<br>
I hope this gets acknowledged by the team, I have a feeling it’s simply a matter of a timeout which needs to be increased.</p>

**Comment by dhalbert** [comment]
<p>We are seeing similar, on long-running large workflows we have. The “Summary page” is especially affected. I’ve opened a ticket.</p>

**Comment by cytopia** [comment]
<p>As I’m facing the same issue I’ve opened up a ticket here: <a href="https://github.com/actions/runner/issues/1786" class="inline-onebox" rel="noopener nofollow ugc">HTTP 502 "This page is taking too long to load" on matrix workflow · Issue #1786 · actions/runner · GitHub</a></p>

**Comment by nizzantalon** [comment]
<p>I can update that in a ticket I opened, I got a response from the GH team on Mar25 that the issue should be resolved soon:</p>
<blockquote>
<p>The engineering team plans to ship a fix out for this early next week. If you still experience this issue by the end of next week, please reach back out!</p>
</blockquote>

**Comment by ethomson** [comment, marked as answer]
<p>Apologies for the trouble here. We believe that we’ve rolled out a fix for these problems, but please do let us know if you’re still seeing it.</p>

**Reply by GBrusse** [reply]
```
###Browser times out on install attempt for Workflow Learning App
###Reproducing bug:
1. In browser got to: https://github.com/BoiseState/CS471-Assignments-Workflows
2. In a new tab open the link for WorkflowLearning GitHub App via either blue hyperlink located at step 6
3. From the page: https://github.com/apps/workflowlearning click the green install button on the right side of the page.

###Expected result:
App installs

###Actual result
Time out error (with unicorn)

Desktop: Win11
Browser: Chrome, Firefox, Edge

Laptop: iOS
Browser: Chrome, Firefox, Safari
```

**Comment by sarathso** [comment]
<p>Yes, it works now and Thank you !!!</p>
